charts/opensearch/CHANGELOG.md

```diff
@@ -13,6 +13,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 ### Security
 ---
+## [2.4.0]
```

Comment:
This needs to become `2.4.2` as there is a PR #279 which is taking up the previous version